sklearn/preprocessing/tests/test_common.py

```diff
@@ -53,3 +55,59 @@ def test_missing_value_handling(est):
                 X_test[:, [i]][~np.isnan(X_test[:, i])])
             assert_array_equal(Xt_col_nonan,
                                Xt_col[~np.isnan(Xt_col.squeeze())])
+
+
+@pytest.mark.parametrize(
+    "est",
+    [QuantileTransformer(n_quantiles=10, random_state=42)]
+)
+@pytest.mark.parametrize(
+    "func_sparse_format",
+    [sparse.csr_matrix,
+     sparse.csc_matrix]
```

Comment:
This should include all sparse array formats IMO (see e.g. https://github.com/scikit-learn/scikit-learn/blob/7124d87dca2839cc9be2f6d51fec6cb52cc5609a/sklearn/utils/estimator_checks.py#L440)